std/parallelism.d

```diff
@@ -2864,13 +2864,13 @@ public:
             }
         }
 
-        ref T opIndex(size_t index)
+        ref T opIndex(size_t index) const
```

Comment:
wait, that can't be right. You shouldn't get a ref to a mutable T from a const range.